token-lending/program/src/instruction.rs

```diff
@@ -219,7 +219,8 @@ pub enum LendingInstruction {
     BorrowObligationLiquidity {
         /// Amount of liquidity to borrow - u64::MAX for 100% of borrowing power
         liquidity_amount: u64,
-        // @TODO: slippage constraint - https://git.io/JmV67
+        /// Minimum amount of liquidity to receive, if borrowing 100% of borrowing power
+        slippage_limit: u64,
     },
```

Comment:
How about separating this into two instructions? We've got `BorrowObligationLiquidity` to specify exactly how much to borrow, and then a new instruction like `BorrowAllObligationLiquidity` which uses 100% of borrowing power and specifies just a slippage.  In the implementation, you call out to the same processor function, but this allows for clearer params, similar to how `Transfer` and `TransferChecked` work in the token program. What do you think?